.github/workflows/aggkit-e2e-single-chain.yml

```diff
@@ -0,0 +1,176 @@
+name: Configurable E2E Execution
```

Comment:
Copied from agglayer/aggkit.
Modified to add

- kurtosis-cdk-repo
- kurtosis-cdk-ref
- build cdk-erigon docker image before running kurtosis
- extract cdk-erigon docker image tag from kurtosis-cdk-args